packages/lib/src/validations/VanillaContracts.ts

```diff
@@ -0,0 +1,19 @@
+import Contract from '../artifacts/Contract';
+import ContractAST from '../utils/ContractAST';
+
+let VANILLA_CONTRACTS = '@openzeppelin/contracts/';
+
+export function importsVanillaContracts(contract: Contract, buildArtifacts?: any): string[] | null {
+  const ast = new ContractAST(contract, buildArtifacts, { nodesFilter: ['ContractDefinition'] });
+  const illegalImports = [...ast.getImports()]
+    .filter(i => i.startsWith(VANILLA_CONTRACTS))
+    .map(i => i.slice(VANILLA_CONTRACTS.length))
+    .map(i => i.replace(/^contracts\//, ''));
```

Comment:
The `@openzeppelin/contracts` package doesn't have the `contracts` directory in it. This line shouldn't be necessary.

Reply:
I know, but since it doesn't hurt, I preferred to leave it here in case we change our minds (again) regarding the folder structure.